Sources/HiPMobile/DataAccessLayer/DbContextDebugView.cs

```diff
@@ -0,0 +1,284 @@
+﻿using Microsoft.EntityFrameworkCore.ChangeTracking;
+using Microsoft.EntityFrameworkCore.Metadata;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.EntityFrameworkCore
+{
+    /// <summary>
+    /// Provides properties that allow the developer to drill into the current
+    /// database data and change tracking information during a debug session.
+    /// </summary>
+    [DebuggerDisplay("🔧 Inspect data and tracked entities")]
```

Comment:
Great usage of Unicode support in C#, highly improves the quality of our codebase. Thanks! :laughing: